cmd/crc/cmd/start.go

```diff
@@ -42,7 +42,7 @@ func runStart(arguments []string) {
 		errors.Exit(1)
 	}
 
-	preflight.StartPreflightChecks()
+	preflight.StartPreflightChecks(crcConfig.GetString(config.VMDriver.Name))
```

Comment:
a `struct` to be passed in is more appropriate, as this would be in line with the rest of the functionality.

we already do this for the start command and therefore the same start struct COULD be mapped or even used.